ctf/final-ctf/challenge-folder-1/deploy/Makefile

```diff
@@ -1,19 +1,6 @@
-FILE	:= ../flag
-FLAG	:= $(shell cat $(FILE))
+EXTERNAL_PORT := 8081
```

Comment:
Not sure why we use `:=`, since it's not really required. We can just use `=`. Is this some sort of best practice?

Reply:
It is. `:=` variables are only expanded once, whereas `=` variables are expanded recursively, each time they are used. Check [this link](https://stackoverflow.com/questions/4879592/whats-the-difference-between-and-in-makefile#:~:text=Expanded%20assignment&text=%3D%20defines%20a%20recursively%2Dexpanded%20variable,defines%20a%20simply%2Dexpanded%20variable.) for an example. But it's a really minor thing in the case of simple variables.